projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -533,6 +540,63 @@ function wpcom_track_publish_first_post_task() {
 	wpcom_mark_launchpad_task_complete_if_active( 'first_post_published_newsletter' );
 }
 
+/**
+ * Gets the number of published posts that were not created by Headstart.
+ *
+ * @return int
+ */
+function wpcom_launchpad_get_published_non_headstart_posts_count() {
+	// Ensure that Headstart posts don't affect the count
+	if ( defined( 'HEADSTART' ) && HEADSTART ) {
+		return 0;
+	}
+
+	// Make sure we don't count any published posts that were created by Headstart.
+	$headstart_posts_filter = array(
+		'post_type'    => 'post',
+		'fields'       => 'ids',
+		'status'       => 'publish',
+		'meta_key'     => '_hs_old_id',
+		'meta_compare' => 'EXISTS',
+	);
+
+	$total_posts_count     = wp_count_posts( 'post' )->publish;
+	$headstart_posts_count = count( get_posts( $headstart_posts_filter ) );
```

Comment:
Noting for posterity -- it looks like this query should be reasonably quick in general as we can use the `type_status_date` index to identify rows that meet the `post` and `publish` conditions quite quickly, and then the secondary query to join against the `post_meta` is relatively direct and quick. This should also be cacheable, so will be generally performant across sites.